src/components/elements/image-with-direction.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import styled from 'styled-components'
+import { useIsRtl } from 'components/hooks/use-isrtl'
+
+export type TImageWithDirection = {
+    is_rtl: boolean
```

Comment:
beside `is_rtl` can we add necessary types for image such as `alt` `width` `height` to be sure everyone is using these props as well while using `ImageWithDirection` component? If typescript already throws an error when we didn't pass those props so please skip this comment.

Reply:
I think it's not cool to force people to pass `width`, `alt`, the type checking for component props works. so it's already there.